node/src/components/linear_chain_sync.rs

```diff
@@ -398,16 +310,27 @@ where
         match event {
             Event::Start(init_peer) => {
                 match self.state {
-                    State::None | State::Done | State::SyncingDescendants { .. } => {
+                    State::None | State::Done => {
                         // No syncing configured.
                         trace!("Received `Start` event when in {} state.", self.state);
                         Effects::new()
                     }
                     State::SyncingTrustedHash { trusted_hash, .. } => {
-                        trace!(?trusted_hash, "Start synchronization");
+                        trace!(?trusted_hash, "Start synchronization until trusted hash.");
                         // Start synchronization.
                         fetch_block_by_hash(effect_builder, init_peer, trusted_hash)
                     }
+                    State::SyncingDescendants {
+                        ref latest_block, ..
+                    } => {
+                        let trusted_hash = latest_block.hash();
+                        trace!(
+                            ?trusted_hash,
+                            "Start synchronization descendants of the trusted block."
```

Comment:
```suggestion
                            "Start synchronizing descendants of the trusted block."
```
(Also, I believe the logs should be lowercase and without a period, at least according to a [_proposed_ standard](https://casperlabs.atlassian.net/wiki/spaces/REL/pages/16842753/Coding+Standards).)

Reply:
Done in [`862d9ed` (#731)](https://github.com/CasperLabs/casper-node/pull/731/commits/862d9edb286a91e18e8d2981fed0d6a74835c3ae)